File: muridku/muridku.backend/GlobalDbProcessor/Muridku.QueryRequestReceiver/Models/Dbs/KtbMember.cs
namespace Muridku.QueryRequestReceiver.Models.Dbs
{
  public class KtbMember : ModelDbBase
  {
    public long ktb_id { get; set; }
    public long member_id { get; set; }
    public int is_pktb { get; set; }
    public int is_active { get; set; }
    public long? old_ktb_id { get; set; }
  }
}

File: muridku/muridku.web/app/Models/KtbMember.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;
use Illuminate\Notifications\Notifiable;

class KtbMember extends Model
{
    use HasFactory, Notifiable;

    protected $table = 'ktbmember';
    const CREATED_AT = 'dtm_crt';
    const UPDATED_AT = 'dtm_upd';

    protected $fillable = [
        'meet_dt',
        'material_name',
        'material_chapter',
    ];

    protected $hidden = [
        'ktb_id',
        'member_id',
        'is_pktb',
        'is_active',
        'old_ktb_id',
        'dtm_crt',
        'usr_crt',
        'dtm_upd',
        'usr_upd'
    ];
}
